diagnostic.md

```diff
@@ -150,7 +186,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will be on the right of the page
+divTwo, divThree, divFive will be all next to each other on the left of the page
+divFour will be in the first (and only) item in the 2nd row
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```